gc/verbose/VerboseHandlerOutput.cpp

```diff
@@ -216,7 +207,7 @@ MM_VerboseHandlerOutput::getTagTemplateWithOldType(char *buf, uintptr_t bufsize,
 }
 
 uintptr_t
-MM_VerboseHandlerOutput::getTagTemplate(char *buf, uintptr_t bufsize, uintptr_t id, const char *type, uintptr_t contextId, uint64_t timeus, uint64_t wallTimeMs)
+MM_VerboseHandlerOutput::getTagTemplate(char *buf, uintptr_t bufsize, uintptr_t id, const char *type, uintptr_t contextId, uint64_t timeus, uint64_t wallTimeMs, const char *reasonForTermination)
 {
```

Comment:
I guess we could factor one more step with parameters list `char *buf, uintptr_t bufsize, uintptr_t id, const char *type, uintptr_t contextId,` with line 178. Not sure it is necessary.